src/main/java/org/broadinstitute/hellbender/tools/SplitNCigarReads.java

```diff
@@ -117,7 +117,7 @@ protected Object doWork() {
 
     private SAMFileGATKReadWriter initialize(final SamReader in) {
         final SAMFileHeader outputHeader = ReadUtils.cloneSAMFileHeader(in.getFileHeader());
-        final SAMFileGATKReadWriter outputWriter = new SAMFileGATKReadWriter(ReadUtils.createCommonSAMWriter(OUTPUT, REFERENCE_SEQUENCE, outputHeader, true, false, false));
+        final SAMFileGATKReadWriter outputWriter = new SAMFileGATKReadWriter(ReadUtils.createCommonSAMWriter(OUTPUT, REFERENCE_SEQUENCE, outputHeader, false, false, false));
```

Comment:
Add a comment explaining why `preSorted` is set to false here, and preserve the warning about deep coverage from GATK3: `"we don't want to assume that reads will be written in order by the manager because in deep, deep pileups it won't work"`